docs/flashbots-auction/advanced/rpc-endpoint.mdx

```diff
@@ -869,6 +869,167 @@ If the first address matches the authentication signature, then a response with
 
 If the signature is invalid or does not match the first address, an appropriate error will be returned instead.
 
+### MEV Refund JSON-RPC API Documentation
```

Comment:
I would delete this section (lines 872-874) and instead repeat the context on line 874 (APIs are for MEV-Share, not authenticated) in each individual API entry below. The reason is:
* It's not standard to have sub-headings in our API docs today (we can revisit this, but it doesn't match the rest of the docs so would prefer to keep things consistent for now)
* We want all of the context to be mentioned in each entry because a user might navigate directly to one of the API methods and miss this overall heading (in which case it won't be clear that the API they're looking at is for MEV-Share or does not require authentication)